libraries/core-react/src/components/EdsProvider/EdsProvider.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import styled from 'styled-components'
+import { EdsProvider } from '../..'
+import * as stories from './EdsProvider.stories.tsx'
+
+<Meta title="Components/EdsProvider"
+  component={EdsProvider}
+  args={{  density: 'comfortable' }} />
```

Comment:
I don't think you need to set the explicitly? Should be derived from default value of props.